portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,25 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Home</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
+    <div class="topnav">
```

Comment:
Consider using the more semantic nav element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

(The top example isn't your use case, but nav is used for showing navigation links in general.)